modules/core/src/debug/loggers.js

```diff
@@ -0,0 +1,130 @@
+const logState = {
+  amMessages: []
```

Comment:
Nit: Abbreviated variable names... had to scan the file to understand what  `am` means.